keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/ScalarDoubleTensor.java

```diff
@@ -597,6 +602,21 @@ public DoubleTensor floorInPlace() {
         return this;
     }
 
+    @Override
+    public DoubleTensor roundInPlace() {
+        double valueToRound = value;
+        if (value < 0. && value + 0.5 == (double) value.intValue()) {
+            // This is necessary to match Python behaviour
```

Comment:
Might want to pop this at the top as Javadoc so this possibly unexpected behaviour is visible for all to see.